lerna.json

```diff
@@ -6,5 +6,10 @@
     "integration/*",
     "demo/*"
   ],
+  "commands": {
+    "bootstrap": {
+      "hoist": "**"
```

Comment:
Should we hoist cloud packages only? I'm not sure if we will be able to generate proper shrinkwrap for demo server.